src/installation/README.md

```diff
@@ -0,0 +1,185 @@
+# Installfest: From zero to hero
+
+## Introduction
+We understand that setting up your computer for development can be a daunting task. There are so many tools and libraries that you need to install, and it's hard to know where to start. That's why we created this guide to help you get started with your development environment.
+
+![Lost Kermit](./lost-kermit.gif)
+
+This guide will help you install all the tools you need to start coding. It will also help you configure your computer so that you can easily switch between different versions of Node.js and Python. Your don't need to install everything in this guide, only the tools you need for your project. If you are not sure what tools you need, ask your project lead or one of the exec members.
+
+## Table of Contents
+  - [Installing Homebrew](#installing-homebrew)
+  - [Installing Git](#installing-git)
+  - [Installing Node](#installing-node)
+  - [Installing Nvm](#installing-nvm)
+  - [Installing Python](#installing-python)
+  - [Installing Pip](#installing-pip)
+  - [Installing Pyenv](#installing-pyenv)
+  - [That's it!](#thats-it)
+
+## Installing Homebrew
+Homebrew is a package manager for macOS (or Linux) that allows you to easily install and manage software packages and libraries. It simplifies the installation process by automating the installation of dependencies and providing a consistent interface for installing software. To install Homebrew, run the following command:
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+To check if Homebrew is installed, run the following command:
+
+```bash
+brew --version
+```
+
+It should return something like this:
+
+```bash
+Homebrew 3.2.0
+Homebrew/homebrew-core (git revision 3b6; last commit 2021-07-26)
+```
+
+## Installing VSCode
```

Comment:
Can we add a small list of generally recommended plugins for our projects like prettier and eslint?

Reply:
The goal of Installfest is to help new members with the initial setup. I think Eslint/Prettier guides should be added to the project guides, not here. It depends on a case-by-case basis 🤔. PMs using React won't need the same plugins as the ones using Vue.js, Node, or any other stack. Also, it's worth noting that some Project CLIs, like NextJS, come with their own plugins, and project members might want to stick with the default ones 🤷‍♂️.

Reply:
Good point, we do already recommend a couple plugins for PMs to consider in our Project Manager Guide. I created another issue to add it to our docs site too. https://github.com/techstartucalgary/Docs/issues/19

Thanks!